.jvmopts

```diff
@@ -0,0 +1,3 @@
+-XX:+IgnoreUnrecognizedVMOptions
```

Comment:
In that file some options only apply to specific versions of JVM and by default if you pass an option onto the JVM that doesn't exist it will immediately terminate. `--illegal-access=permit` only works on JVM 16 and `--add-opens java.base/java.lang=ALL-UNNAMED` only works on JVM 17

Reply:
May be we could do better?
 - provide a runnable script which properly assemble command line options
 - adopt https://www.elastic.co/guide/en/elasticsearch/reference/current/advanced-configuration.html#jvm-options-syntax

Why I am concerned: we should not silence invalid JVM command line options / flags, the users should be aware of any mismatches here.

Reply:
I thought of this originally but the issue here is the typical way/s that SBT is used. For example a lot of people (myself included) use Intellij's SBT integration when working with Scala projects. When using sbt like this the only possible way to pass JVM arguments is with `.jvmopts`. The same applies to the official sbt launcher script which is included when you install sbt via your operating systems package manager (homebrew, apt, pacman etc etc).

Ideally sbt should have support for dynamically supporting different JVM options depending the JVM option but this doesn't exist yet.

Reply:
:+1: all set!